docs/v3.x/concepts/models.md

```diff
@@ -4,79 +4,86 @@
+  "collectionName": "restaurants",
   "info": {
-    "name": "restaurant",
+    "name": "Restaurant",
     "description": "This represents the Restaurant Model"
   },
+  "options": {
+    "increments": true,
+    "timestamps": true
+  },
   "attributes": {
-    "cover": {
-      "collection": "file",
-      "via": "related",
-      "plugin": "upload"
-    },
     "name": {
-      "default": "",
       "type": "string"
     },
     "description": {
-      "default": "",
       "type": "text"
+    },
+    "cover": {
+      "collection": "file",
+      "via": "related",
+      "plugin": "upload"
     }
   }
 }
 ```
 
-In this example, there is a `Restaurant` model which contains the attributes `cover`, `name` and `description`.
+In this example, there is a `restaurant` model which contains the attributes `name`, `description` and `cover`.
 
 ### Component's models
 
-It also exist another type of models named `components`. A component is a data structure that can be used in one or many other API's model. There is no lifecycle related, only a JSON file definition
+A Component is a self contained database structure that can be reused in one or more Content Type. There is no lifecycle related, only a JSON file definition.
 
-**Path —** `./components/default/simple.json`
+**Path —** `/components/default/meta.json`.
 
 ```json
 {
-  "connection": "default",
-  "collectionName": "components_default_simples",
+  "collectionName": "components_seo_metas",
   "info": {
-    "name": "simple",
+    "name": "Meta",
     "icon": "arrow-circle-right"
   },
   "options": {},
   "attributes": {
-    "name": {
+    "title": {
+      "type": "string"
+    },
+    "description": {
       "type": "string"
     }
   }
 }
 ```
 
-In this example, there is a `Simple` component which contains the attributes `name`. And the component is in the category `default`.
+In this example, there is a `Meta` component which contains the attributes `title` and `description` which are of type text. For better organization and easier attachment multiple components to a Content Type they are stored within categories. Above one lives in the `seo` category.
 
 ### Where are the models defined?
 
-For **Content Types**, models are defined in each `./api/**/models/` folder. Every JavaScript or JSON file in these folders will be loaded as a model. They are also available through the `strapi.models` and `strapi.api.**.models` global variables. Usable everywhere in the project, they contain the ORM model object that they refer to. By convention, a model's name should be written in lowercase.
+For **Content Types**, models are defined in each `/api/**/models/` folder. Every JavaScript or JSON file in these folders will be loaded as a model. They are also available through the `strapi.models` and `strapi.api.**.models` global variables. Usable everywhere in the project, they contain the ORM model object that they refer to. By convention, a model's name should be written in lowercase.
```

Comment:
`/api...` is not valid when writing file paths either `./api` or `api/` but `/api` would mean that the the api folder is on the root of the file system (looking at it from a linux perspective)